test/known_issues/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,7 +15,7 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[16, 18, 20].forEach((exponent) => {
+[16, 18, 20, 21, 22].forEach((exponent) => {
```

Comment:
22 is the biggest integer you can use before you start getting `RangeError` failures for `Invalid string length`.